migration/v2/model/identity.go

```diff
@@ -5,12 +5,11 @@ import (
 	"github.com/konveyor/tackle2-hub/encryption"
 )
 
-//
 // Identity represents and identity with a set of credentials.
 type Identity struct {
 	Model
 	Kind        string `gorm:"not null"`
-	Name        string `gorm:"not null"`
+	Name        string `gorm:"index;unique;not null"`
```

Comment:
@mansam This probably not the best place for the modification, may I ask for your input?
- should the identity.go model be just created with this change in v4 directory?
- if there is a migration function for this change, assuming it should check existing identities in database and modify name if duplicate? Thanks!

Reply:
> @mansam This probably not the best place for the modification, may I ask for your input?
> 
> * should the identity.go model be just created with this change in v4 directory?

Yes.

> * if there is a migration function for this change, assuming it should check existing identities in database and modify name if duplicate? Thanks!

Yes.  You'll want to add logic in the v4/migrate.go to find dups and add a suffix (or something).  This will need to happen before the auto-migrate is called.